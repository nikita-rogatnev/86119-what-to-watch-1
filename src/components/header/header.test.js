import React from 'react';
import renderer from 'react-test-renderer';

import Header from './header';

jest.mock(`./header`, () => `Header`);

it(`Header renders correctly`, () => {
  const tree = renderer
    .create(<Header
      isLogged={false}
      isAuthorizationRequired={false}
      requireAuthorization={() => jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
