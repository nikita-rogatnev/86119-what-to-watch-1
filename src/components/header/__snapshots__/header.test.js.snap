// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<Header
  isAuthorizationRequired={false}
  isLogged={false}
  requireAuthorization={[Function]}
/>
`;
